### DISCUSSION TITLE
any body use twitter Notifications ?

### DISCUSSION THREAD

**Discussion by osgoodtr** [mcguirepr89/BirdNET-Pi#1079] [Q&A]
hi guys

i want to tweet to twitter. i follow to apprise twitter introduction 'https://github.com/caronc/apprise/wiki/Notify_twitter'
 but twitter developer is change. this introduction doesnt work. how to set up "User authentication settings"? what is "Callback URI / Redirect URL"?
if i follow twitter introduction like this link https://github.com/caronc/apprise/wiki/Notify_twitter . birdnet error like this.

2023-11-30 19:59:31,735 - INFO - Notifying 1 service(s) asynchronously. 2023-11-30 19:59:32,190 - WARNING - Failed to send Twitter POST to https://api.twitter.com/1.1/statuses/update.json: error=403.

**Comment by KB3LZV** [comment]
I can't help you, but I hope someone answers. This is something that I want to do.
